easybuild/easyblocks/generic/intelbase.py

```diff
@@ -372,8 +389,9 @@ def make_module_extra(self):
         """Custom variable definitions in module file."""
         txt = super(IntelBase, self).make_module_extra()
 
-        txt += self.module_generator.prepend_paths(self.license_env_var, [self.license_file],
-                                                   allow_abs=True, expand_relpaths=False)
+        if self.requires_runtime_license:
```

Comment:
we need to think about what this does under `--module-only`, although it may be OK since `prepare_step` isn't skipped then?